java/java.mx.project/test/unit/src/org/netbeans/modules/java/mx/project/SuiteActionProviderTest.java

```diff
@@ -91,7 +91,7 @@ public void finished(boolean success) {
         ap.invokeAction(ActionProvider.COMMAND_BUILD, ctx);
 
         assertTrue("Progress started", progress.started);
-        progress.finished.await(45, TimeUnit.SECONDS);
+        progress.finished.await(600, TimeUnit.SECONDS);
```

Comment:
i tried to reproduce the issues with this test locally a few times already. But it always passes. This test is also super fast if i remember correctly. It finishes in 700ms or so.

So I don't think this could be timeout related - but who knows.

Reply:
also as note: the tests are wrapped in a retry script which retries twice on failure. So long timeout could lead to really long test times on failure.

Reply:
The last time it failed with the AssertionError thrown from [SuiteActionProviderTest.java#L95](https://github.com/apache/netbeans/blob/af5a1da6f7b5c826b301d0ac422bcaec5030ea0a/java/java.mx.project/test/unit/src/org/netbeans/modules/java/mx/project/SuiteActionProviderTest.java#L95)

Reply:
yeah. that one fails very often. Yesterday I looked at the last 18 merges to master and wrote down all failures to wrap tests in retry scripts. This test was on top.